openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -694,13 +694,16 @@ Cache-Control: no-store
 
 # Credential Endpoint {#credential-endpoint}
 
-The Credential Endpoint issues a Credential as approved by the End-User upon presentation of a valid Access Token representing this approval. Support for this endpoint is REQUIRED.
+The Credential Endpoint issues one or more Credentials of the same Credential Configuration and Credential Dataset (as approved by the End-User) upon presentation of a valid Access Token representing this approval. Support for this endpoint is REQUIRED.
 
-Communication with the Credential Endpoint MUST utilize TLS. 
+Communication with the Credential Endpoint MUST utilize TLS.
 
-The Client can request issuance of a Credential of a certain type multiple times, e.g., to associate the Credential with different public keys/Decentralized Identifiers (DIDs) or to refresh a certain Credential.
+The Client sends a Credential Request to obtain:
 
-If the Access Token is valid for requesting issuance of multiple Credentials, it is at the Client's discretion to decide the order in which to request issuance of multiple Credentials requested in the Authorization Request.
+* one Credential; or
+* multiple Credential instances of the same Credential Configuration and Credential Dataset, each with distinct cryptographic material.
+
+The Client can send several consecutive Credential Requests to obtain multiple Credentials with the same or different Credential Datasets. If the Access Token allows the request for multiple Credentials, the Client has the flexibility to determine the sequence in which they are requested.
```

Comment:
this last sentence is a repetition of the last point. We should remove this repetition and include the following sentece

````
it is important to
note that the ability to do so depends on whether the Issuer supports the multiple
issuance of the same Credential Type or the refresh of the Credential.
````

Reply:
please check the proposal